HealthSDK/GiniHealthSDKExample/GiniHealthSDKExample/ComponentAPICoordinator.swift

```diff
@@ -379,7 +379,7 @@ extension ComponentAPICoordinator: UINavigationControllerDelegate {
         if toVC is CameraViewController &&
             (fromVC is ReviewViewController ||
              fromVC is AnalysisViewController ||
-             fromVC is ImageAnalysisNoResultsViewController) {
+             fromVC is HelpTipsViewController) {
```

Comment:
Please, 🙏 revert this change, we don't touch HealthSDK now ;)

Reply:
@zladzeyka ok, I will revert this change and old NoResults screen too

Reply:
@chriskryn thank you 😊 🙏